app/players/dwarves/dwarf.py

```diff
@@ -0,0 +1,11 @@
+from app.players.player import Player
+from abc import ABC
+
+
+class Dwarf(Player, ABC):
```

Comment:
You don’t need ABC inheritance here as your class is not realises any abstract methods

Reply:
Thanks! Done